kustomize/overlays/stable/secrets/keycloak-secrets.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   name: keycloak-secrets
 spec:
   encryptedData:
+    KEYCLOAK_USER_ID: AgBqr9S/CBggW4OIXgo8wRuF9CpuOsIxPNqaure6XztYmGqNlTTQKrSw0gZQgjy7rDWkffeC8q/nwXsPhaQLU9qaEGzu9mBnQUi7N7aq06QpNx4HLXmDYfhwB97ULqVViRy9tmStVknlL3CAaT/K8oXVoqAcOSB5sSeLDH5eCIyf4iW7PfpOSPK5zZPXoRahd15r/YqGOzLjSsC8KRise6o7oDYdIcBL0GdvrslXuCEF5s9lsMVRe2ZTmKQg9OD2WG+jdgRlS4ryUnhWDgWSpHTo9iCL6vY+kUZ/lTC7gBSxoplxaYWeOGgwfeooFI2Fc2euqi3m8JmKlAQ2FOkC8ij84gHlv7sCoTTA/+HVQwd8yiIDmqFyjneswbWSGTzcjy4motoSnrFxvKYVo3BfO6ypYxMt3sh7T1TtQGgyqOLIyNJlNF5lcDDAIalOwFLure0nDy9yF4SQ/q2BXAGZM/3HxVjdBb/SgJcStJ3/a+3fh0aeN9X3qlhyJLtTqc616cgQEz2BuV9bYfW3Eq5hNPjcteVaXbA6lzZZgFGof6hpW/NK14n3BThMJfvRHc1hwrxvhzRK9kirzra5X3Wuw5TZHNo3pNtPEm+ybqDesjtqEHLl8k2dR4S1pndGTY6Uq5zETOZZjNYyGoEVdqMfkBKQUXJ6SiwOm7pZEv2dDXaaQEPDUCaH3gui1JSCrDedojP/xZ+mU/eJ0L/uXO4ZrIFqxXYleMdvOaE/vYEMRHjkTccjHq0=
```

Comment:
This string is exactly the same as the one in the `prod` overlay, but it can't work this way because the two environments use different encryption keys.
You must regenerate the secret using the `stable` encryption key instead of copy the one from `prod`.

Reply:
@kostola The update to our `kustomize/README.md` is part of https://issues.redhat.com/browse/MGDOBR-331, which is assigned to you. Can you do an update to the README.md to provide the instructions on which key to use so that the rest of the team is aware of the need to do that.

cheers,

Rob

Reply:
Updated `stable` encryption key